sdk/documenttranslator/ai-document-translator-rest/src/paginateHelper.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import type { PagedAsyncIterableIterator, PagedResult } from "@azure/core-paging";
```

Comment:
I remember we don't need this core-paging now.

Reply:
we need release a new version to support remove core-paging dep